File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/01_First_Steps_In_Coding_Exercise/08_Basketball_Equipment.js
function basketballEquipment(input) {
    let annualFee = Number(input[0]);

    let shoes = annualFee * (100 - 40) / 100;
    let equip = shoes * (1 - 0.2);
    let ball = equip / 4;
    let accessoirs = ball / 5;

    let totalCosts = annualFee + shoes + equip + ball + accessoirs;

    console.log(totalCosts);
}

// basketballEquipment(["550 "]);

File: Python_Courses/Python_Basics/Python_PB_2025_03/01_First_Steps_in_Coding_Exercise/08_Basketball_Equipment.py
# 1. E
annual_fee = int(input())

# 2. T
sneakers = annual_fee * 0.60  # annual_fee * (1 - 0.40)
equip = sneakers * 0.80  # sneakers * (1 - 0.20)
ball = equip * 0.25  # equip * 1 / 4
accessoirs = ball * 0.20  # ball * 1 / 5

final_price = annual_fee + sneakers + equip + ball + accessoirs

# 3. L
print(final_price)
